allennlp/commands/serve.py

```diff
@@ -9,10 +13,17 @@ def add_subparser(parser: argparse._SubParsersAction) -> argparse.ArgumentParser
 
     subparser.add_argument('--port', type=int, default=8000)
     subparser.add_argument('--workers', type=int, default=1)
+    subparser.add_argument('--configfile', type=argparse.FileType('r'), default=None, help="path to a JSON file specifying the configuration for the models")
```

Comment:
How do I read this setting if I name it `config-file`?

Reply:
behind the scenes it should get saved as `args.config_file`